### ISSUE TITLE
Enforce 2 spaces of indentation

### ISSUE THREAD

**Issue by alexyuly** [Intelight/tslint-intelight#16]
Apparently TSLint does not currently support enforcing a certain number of spaces of indentation... see [here](https://github.com/palantir/tslint/issues/2814).

TypeScript makes me rage more and more every day.

@TimMikeladze

**Comment by TimMikeladze** [comment]
Yeah that's not ideal. I think we should setup prettier to be ran as part of our linting process to make up for any tslint inconsistencies

**Comment by alexyuly** [comment]
Yeah, prettier seems to be the standard solution people are using for this problem.

**Comment by alexyuly** [comment]
Can we use _only_ prettier, so we don't have to rely on _two_ linting tools? It seems like we could use just prettier on its own and be done with this.
